manifests/config/exporter/exporter.yaml

```diff
@@ -94,6 +95,8 @@ spec:
         - name: redfish
           mountPath: /etc/redfish
           readOnly: true
+        - name: qat
+          mountPath: /usr/local/bin/adf_ctl
```

Comment:
is `adf_ctl` available in rpm? We typically package utility binary such as [cpuid](https://github.com/sustainable-computing-io/kepler/blob/main/build/Dockerfile.bcc.base#L7) in the base image

Reply:
adf_ctl is not available in rpm, the binary only exists on servers with QAT installed..

Reply:
In this case, can you make a kustomize option instead of changing the base `exporter.yaml`? You can check the CI patch [here](https://github.com/sustainable-computing-io/kepler/blob/main/manifests/config/exporter/patch/patch-ci.yaml)

cc @jichenjc for info

Reply:
Sure, I will take this approach. Thanks!